src/solver/optimisation/opt_fonctions.h

```diff
@@ -62,13 +61,6 @@ void OPT_InitialiserLesCoutsQuadratiques(PROBLEME_HEBDO*, int);
 void OPT_ControleDesPminPmaxThermiques(PROBLEME_HEBDO*);
 bool OPT_AppelDuSolveurQuadratique(PROBLEME_ANTARES_A_RESOUDRE*, const int);
 
-using namespace Antares::Data::AdequacyPatch;
-bool ADQ_PATCH_CSR(PROBLEME_ANTARES_A_RESOUDRE&,
-                   HourlyCSRProblem&,
-                   const AdqPatchParams&,
-                   uint week,
-                   int year);
```

Comment:
**ADQ_PATCH_CSR** had nothing to do in **opt_fonctions.h**.
Beside, it caused unwanted re-compilation when making changes in **hourly_csr_problem.h** (notice the inclusion of **hourly_csr_problem.h** here).